projects/storefrontlib/src/public_api.ts

```diff
@@ -3,7 +3,6 @@
  */
 
 export * from './cms-components/index';
-export * from './cms-components/myaccount/index';
```

Comment:
Why are we not exporting my account anymore?

Reply:
They already been exported in `/cms-components/index`.